app/src/main/java/com/example/mpweather/MainActivity.kt

```diff
@@ -14,12 +14,13 @@ import com.android.volley.toolbox.Volley
 import org.json.JSONException
 import org.json.JSONObject
 import java.text.DecimalFormat
+import kotlinx.android.synthetic.main.activity_main.*
 
 class MainActivity : AppCompatActivity() {
     //Declare the view object refrences
-    var etCity: EditText? = null
-    var etCountry: EditText? = null
-    var tvResult: TextView? = null
```

Comment:
These variables are not needed because synthetics are taking care of it.

Reply:
Synthetics == good.  More concise code.